### ISSUE TITLE
Crash when R/G/B number value is out of range

### ISSUE THREAD

**Issue by thegatesdev** [UltimateDoge5/PhantomShapes#2]
You can for example try to type '300' in one of the number values for RGB, and it will crash:
```
net.minecraft.class_148: Rendering screen
	at net.minecraft.class_757.method_3192(class_757.java:940) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1327) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:888) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:265) ~[minecraft-1.20.4-client.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.11.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.11.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.11.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:100) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) ~[NewLaunch.jar:?]
Caused by: java.lang.IllegalArgumentException: Color parameter outside of expected range: Green
	at java.awt.Color.testColorValueRange(Color.java:312) ~[?:?]
	at java.awt.Color.<init>(Color.java:397) ~[?:?]
	at java.awt.Color.<init>(Color.java:371) ~[?:?]
	at org.pkozak.screen.ShapeEditorScreen.drawColorBox(ShapeEditorScreen.kt:472) ~[phantomshapes-1.0.6.jar:?]
	at org.pkozak.screen.ShapeEditorScreen.method_25394(ShapeEditorScreen.kt:345) ~[phantomshapes-1.0.6.jar:?]
	at net.minecraft.class_437.method_47413(class_437.java:110) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.mixinextras$bridge$method_47413$284(class_757.java) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.wrapOperation$bnl000$fabric-screen-api-v1$onRenderScreen(class_757.java:5111) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:931) ~[client-intermediary.jar:?]
```

**Comment by UltimateDoge5** [comment]
Thanks for the feedback!
I'll try push out the next version with the fix in some time. It's an easy fix but right now I'm working on the support for the `1.20.6` and it's a little messy because of the next java version, mappings changes and other dev environment changes.

**Comment by UltimateDoge5** [comment]
Fixed the issue in version `1.0.7` of the mod. Here is the download link of the version on [modrinth](https://modrinth.com/mod/phantomshapes/version/aCvowtSh).
